package.json

```diff
@@ -7,32 +7,56 @@
     "start": "NODE_ENV=dev nodemon ./src/index.js --exec babel-node --",
     "build": "babel src -d dist ",
     "eslint": "eslint --ignore-path .gitignore .",
-    "dev": "nodemon --exec babel-node ./src/index.js",
-    "test": "NODE_ENV=testing nyc mocha ./src/tests/*.js  --timeout 200000 --exit",
-    "coverage": "nyc npm test  && npm run coveralls-coverage"
+    "dev-start": "nodemon --exec babel-node ./src/index.js",
+    "db-seed": "NODE_ENV=testing sequelize db:seed:all",
+    "test": "NODE_ENV=testing npm run dropTables && NODE_ENV=testing npm run db-migrate && npm run db-seed && NODE_ENV=testing nyc --reporter=text --reporter=html --require @babel/polyfill --require @babel/register mocha ./src/tests/*.js  --timeout 200000 --exit",
+    "coverage": "nyc npm test  && npm run coveralls-coverage",
+    "db-migrate": "sequelize db:migrate",
+    "dropTables": "sequelize db:migrate:undo:all",
+    "create-db": "npx sequelize-cli model:generate --name users --attributes firstName:string,lastName:string,email:string,password:string && sequelize db:migrate"
```

Comment:
We have added two fields in the database that are more necessary, you have to take a look at my PR to add them too